tests/generate_packageTest.py

```diff
@@ -0,0 +1,89 @@
+import copy
+import os
+import sys
+import unittest
+from unittest.mock import patch
+from bears.generate_package import (touch, create_file_from_template,
+                                    create_file_structure_for_packages,
+                                    perform_upload, main,
+                                    create_upload_parser)
+
+
+class touchTest(unittest.TestCase):
+
+    def test_file_doesnt_exist(self):
+        if os.path.exists('TestFile.py'):
+            os.remove('TestFile.py')
+        self.assertFalse(os.path.exists('TestFile.py'))
+        touch('TestFile.py')
+        self.assertTrue(os.path.exists('TestFile.py'))
```

Comment:
Ideally, you would be creating two test method here, covering both cases. One for file exists, one for file does not exist.